common/app/views/fragments/header.scala.html

```diff
@@ -19,13 +19,22 @@
                         <div class="brand-bar__item brand-bar__item--profile
                                     brand-bar__item--has-control brand-bar__item--register
                                     popup-container has-popup js-profile-register">
-                            <a href="@Configuration.id.url/register?skipConfirmation=true&INTCMP=HEADER_GU_REG&returnUrl=@URLEncode("https://membership.theguardian.com/join/friend/enter-details")"
-                               data-link-name="Register link"
-                               class="brand-bar__item--action">
+                        @if(mvt.ABNewFreeMembershipTest.isParticipating) {
+                            <a href="http://membership.theguardian.com"
+                            data-link-name="Register link"
+                            class="brand-bar__item--action">
                                 @fragments.inlineSvg("profile-add-36", "icon", List("rounded-icon", "control__icon-wrapper"))
+                                <span class="control__info js-control-info control__info--supporting">become a member</span>
+                            </a>
+                        } else {
+                            <a href="@Configuration.id.url/register?skipConfirmation=true&INTCMP=HEADER_GU_REG&returnUrl=@URLEncode("https://membership.theguardian.com/join/friend/enter-details")"
```

Comment:
Is it worth adding a campaign code to the https://membership.theguardian.com/join/friend/enter-details returnUrl too? I think we loose some analytics on our end without it (I know we can probably get this data through Omniture in the end too)